src/tyr/route_serializer_osrm.cc

```diff
@@ -316,6 +302,10 @@ std::vector<PointLL> simplified_shape(const valhalla::DirectionsRoute& direction
 void route_geometry(json::MapPtr& route,
                     const valhalla::DirectionsRoute& directions,
                     const valhalla::Options& options) {
+  if (options.shape_format() == no_shape) {
+    return;
+  }
```

Comment:
keep or revert? it's not documented for /route anyways;)

Reply:
i th ink its good to keep